v22.2/alter-backup-schedule.md

```diff
@@ -0,0 +1,190 @@
+---
+title: ALTER BACKUP SCHEDULE
+summary: Use the ALTER BACKUP SCHEDULE statement to change an existing backup schedule.
+toc: true
+docs_area: reference.sql
+---
+
+{{site.data.alerts.callout_info}}
+Core users can only use backup scheduling for [full backups](create-schedule-for-backup.html#create-a-schedule-for-full-backups-only-core) of clusters, databases, or tables.
+
+To use the other backup features, you need an [Enterprise license](enterprise-licensing.html).
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v22.2" %} The `ALTER BACKUP SCHEDULE` statement modifies an existing [backup schedule](manage-a-backup-schedule.html). You can use `ALTER BACKUP SCHEDULE` to do the following:
+
+- Set a different name for a backup schedule.
+- Change a scheduled backup's storage location.
+- Apply additional backup options or schedule options to backups and schedules.
+- Adjust the cadence and type of scheduled backups.
+
+## Required privileges
+
+To alter a backup schedule, you must be the owner of the backup schedule, i.e., the user that [created the backup schedule](create-schedule-for-backup.html).
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/master/grammar_svg/alter_backup_schedule.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+-------------
+`schedule_id` | The schedule's ID that `CREATE SCHEDULE FOR BACKUP` and `SHOW SCHEDULES` display.
+`schedule_label` | The name or label given to the backup schedule.
+`collectionURI` | The URI where you want to store the backup. See [Backup file URLs](backup.html#backup-file-urls) for detail on forming the URI.
+`option` | Control the backup behavior with a comma-separated list of these [options](#backup-options).
+`RECURRING crontab` | Specify when the backup should be taken. By default, these are incremental backups. A separate schedule may be created automatically to write full backups at a regular cadence, depending on the frequency of the incremental backups. You can likewise modify this separate schedule with `ALTER BACKUP SCHEDULE`. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron). All times in UTC. <br><br>Example: `'@daily'` (run daily at midnight)
+`FULL BACKUP crontab / ALWAYS` | Specify when to take a new full backup. Define the schedule as a `STRING` in [crontab format](https://en.wikipedia.org/wiki/Cron) or as `ALWAYS`. <br><br>`FULL BACKUP ALWAYS` will trigger `RECURRING` to always take full backups. **Note:** If you do not have an Enterprise license then `ALWAYS` is the only accepted value of `FULL BACKUP`. <br><br>If you omit the `FULL BACKUP` clause, the default backup schedule will be as follows: <ul><li>`RECURRING` <= 1 hour: Default to `FULL BACKUP '@daily'`</li><li>`RECURRING` <= 1 day: Default to `FULL BACKUP '@weekly'`</li><li>Otherwise: Default to `FULL BACKUP ALWAYS`</li></ul>
```

Comment:
nit: recommend using `if` to highlight three alternate cases:

```
- If `RECURRING` is set to <= 1 hour: Default to `FULL BACKUP` '@daily'
- If `RECURRING` is set to <= 1 day: Default to `FULL BACKUP` '@weekly'
- Otherwise: Default to `FULL BACKUP ALWAYS`
```

Or just:

```
- If `RECURRING` <= 1 hour ... 
```